src/restore_config.c

```diff
@@ -1125,7 +1136,10 @@ usage(const char *name)
 	fprintf(stdout, "                       - Info\n");
 	fprintf(stdout, "                       - Debug\n");
 	fprintf(stdout, "                       - Trace\n");
-	fprintf(stdout, "                      The default is Fatal.\n\n");
+	fprintf(stdout, "                      The default is Fatal.\n");
+	fprintf(stdout, "      --s3-connect-timeout <ms>\n");
+	fprintf(stdout, "                      The internal aws s3 client's connection timeout in milliseconds.\n");
+	fprintf(stdout, "                      Corresponds to the connectTimeoutMS aws-sdk-cpp client configuration.\n\n");
```

Comment:
```suggestion
	fprintf(stdout, "                      The default is Fatal.\n");
	fprintf(stdout, "      --s3-connect-timeout <ms>\n");
	fprintf(stdout, "                      The AWS S3 SDK's connection timeout in milliseconds.\n");
```
The above description uses uppercase and calls it the SDK. This is fine. The client or SDK does not matter to the user.  You don't need to tell them more about what it corresponds to in the actual SDK. If your description does not provide enough info, add it here. Don't make them go looking. They might not be as savvy as we devs

Reply:
I know most users won't know it but this is specific to the Client class in the SDK, that's why I call it "client".
I used the capitalization suggestions, I'd like to keep the mention of the equivalent config items because it lets people look them up for more information.